src/main/java/core/basesyntax/utils/process/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.utils.process.impl;
+
+import core.basesyntax.storage.impl.StorageImpl;
```

Comment:
According to the encapsulation principle, you should not use the specific implementation of Storage (StorageImpl) here. It's better to refer to the interface instead to make your service more flexible and not dependent on a concrete implementation.